packages/vats/src/distributeFees.js

```diff
@@ -60,12 +63,21 @@ export function buildDistributor(treasury, bank, epochTimer, timer, params) {
     if (!queuedPayments.length) {
       return;
     }
-    E(bank).depositMultiple(
-      queuedAccounts.splice(0, depositsPerUpdate),
-      queuedPayments.splice(0, depositsPerUpdate),
-    );
 
-    scheduleDeposits();
+    const accounts = queuedAccounts.splice(0, depositsPerUpdate);
+    const payments = queuedPayments.splice(0, depositsPerUpdate);
+    E(bank)
+      .depositMultiple(accounts, payments)
+      .then(settledResults => {
+        const rejectedPayments = payments.filter(
+          (_pmt, i) =>
+            settledResults[i] && settledResults[i].status === 'rejected',
+        );
+
+        // Redeposit the payments.
+        disposeRejectedPayments(rejectedPayments);
+      });
```

Comment:
This should have a `catch` or should be awaited.

Reply:
I'll add a `catch`.